src/main/kotlin/com/jetbrains/snakecharm/lang/psi/SmkFileReference.kt

```diff
@@ -11,21 +11,24 @@ import com.jetbrains.python.psi.PyStringLiteralExpression
 class SmkFileReference(
         element: PsiElement, textRange: TextRange
 ) : PsiReferenceBase<PsiElement>(element, textRange) {
+    private val key: String = element.text.substring(textRange.startOffset, textRange.endOffset)
+
     override fun resolve(): PsiElement? {
-        if (element !is PyStringLiteralExpression) {
-            return null
-        }
+        val stringLiteral = (element as? SMKWorkflowParameterListStatement)?.
+                argumentList?.children?.firstOrNull {
```

Comment:
` argumentList?.children?.` please don't use this low-level API. See `PyArgumentList` interface methods